toolkit/authentication/src/main/java/com/arcgismaps/toolkit/authentication/Authenticator.kt

```diff
@@ -63,14 +72,78 @@ public fun Authenticator(
     }
 }
 
+/**
+ * Displays appropriate Authentication UI when an authentication challenge is issued.
+ *
+ * For example, when an [ArcGISAuthenticationChallenge] is issued and the [AuthenticatorState] has a corresponding
+ * [OAuthUserConfiguration], then a Custom Tab will be launched to complete the OAuth sign in.
+ *
+ * This Composable will adapt to the size of its container, allowing flexible layout usage. For alternate behavior,
+ * see [DialogAuthenticator].
+ *
+ * @param authenticatorState the object that holds the state to handle authentication challenges.
+ * @param modifier the [Modifier] to apply to this Authenticator.
+ * @param onPendingBrowserAuthenticationChallenge if not null, this will be called when a authentication challenge is pending
```

Comment:
the comment indicates that the lambda is nullable, but it is not.

```suggestion
 * @param onPendingBrowserAuthenticationChallenge a lambda that will be called when an authentication challenge is pending
```

Reply:
I also flagged a small grammar issue later in the sentence... ("an authentication challenge")

Reply:
same suggestions for DialogAuthenticator.